### DISCUSSION TITLE
Can't get 8BitDo Keyboard working on Dreamcast.

### DISCUSSION THREAD

**Discussion by JediStargazer** [darthcloud/BlueRetro#957] [General]
The keyboard seems to pair just fine and it's detected as a device in Typing of the Dead but it wont respond when the game asks me to "press enter"  I've seen previous posts where the keyboard was confirmed to be supported. Perhaps a firmware update on 8bitdo's end broke it?

**Comment by darthcloud** [comment]
I think it's a general BlueRetro problem with that game I'll have to take a look.

**Comment by JediStargazer** [comment]
If it helps at all. I also tested it with Outtrigger and had the same exact issue.

**Comment by darthcloud** [comment]
BTW you need to configure BlueRetro for the dreamcast to see a KB:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#82---keyboard-config-example

If using a single port dongle use "output 1", for multiplayer adapter set it on output 2 as in the gif in above example.

There was an issue with typing of the dead too I got a tentative fix here in this beta FW:
https://github.com/darthcloud/BlueRetro/actions/runs/8762666480

Let me know if you can get it working.
